UICatalog/Scenarios/ColorPicker.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Terminal.Gui;
+using System;
+
+namespace UICatalog.Scenarios {
+	[ScenarioMetadata (Name: "Color Pickers", Description: "ColorPicker demo.")]
```

Comment:
Since the name of the control is "Color Picker", the Name of the scenario should be "Color Picker Demo" not "Color Pickers".

Reply:
Done